script/deploy/mainnet/v0.4.3-upgrade_rewardsCoordinator.s.sol

```diff
@@ -0,0 +1,198 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.12;
+
+import "../../utils/ExistingDeploymentParser.sol";
+import "../../utils/Multisend.sol";
+import "script/utils/TimelockEncoding.sol";
+
+/**
+ * @notice Script used for the first deployment of EigenLayer core contracts to Holesky
+ * anvil --fork-url $MAINNET_RPC
+ *
+ * Holesky testnet: Deploy/Upgrade RewardsCoordinator
+ * forge script script/deploy/holesky/v0.4.3-upgrade-rewardsCoordinator.s.sol --rpc-url $MAINNET_RPC --private-key $PRIVATE_KEY --broadcast -vvvv --verify --etherscan-api-key $ETHERSCAN_API_KEY
```

Comment:
couple typos in here; I believe it should be

forge script script/deploy/mainnet/v0.4.3-upgrade_rewardsCoordinator.s.sol:Upgrade_Mainnet_RewardsCoordinator --rpc-url $MAINNET_RPC --private-key $PRIVATE_KEY --broadcast -vvvv --verify --etherscan-api-key $ETHERSCAN_API_KEY